examples/trials/mnist-batch-tune-keras/config.yml

```diff
@@ -0,0 +1,20 @@
+authorName: default
+experimentName: example_mnist-keras
+trialConcurrency: 1
+maxExecDuration: 1h
+maxTrialNum: 10
+#choice: local, remote
+trainingServicePlatform: local
+searchSpacePath: ~/nni/examples/trials/mnist-batch-tune-keras/search_space.json
+#choice: true, false
+useAnnotation: false
+tuner:
+  #choice: TPE, Random, Anneal, Evolution, BatchTuner
+  builtinTunerName: BatchTuner
+  classArgs:
+    #choice: maximize, minimize
```

Comment:
If optimize_mode will not be used, the classArgs section can be removed.

Reply:
ok

Reply:
The launcher will make sure the tuner must has optimized mode, or it will raise a error. We fixed it later.